src/control/jobs/CustomExportJob.cpp

```diff
@@ -70,7 +72,7 @@ auto CustomExportJob::showFilechooser() -> bool {
 
     Document* doc = control->getDocument();
     doc->lock();
-    auto* dlg = new ExportDialog(control->getGladeSearchPath());
+    auto dlg = std::make_unique<ExportDialog>(control->getGladeSearchPath());
```

Comment:
Is a pointer even necessary here? 
```suggestion
    ExportDialog dlg(control->getGladeSearchPath());
```
(with of course change each `dlg->foo` by a `dlg.foo`)